### PR TITLE
Ako in wcp changes

### PR DESCRIPTION
**Pull request by abhishekbsingh**

### CODE REVIEW

File: internal/nodes/ingress_model_rel.go [hunk 1]
```diff
@@ -986,3 +992,16 @@ func validateSvcApiGatewayForClass(key string, gateway *svcapiv1alpha1.Gateway)
 
 	return nil
 }
+
+func t1LRNSToGateway(t1LR, namespace, key string) ([]string, bool) {
+	allGateways := make([]string, 0)
+	gateways, err := lib.AKOControlConfig().AdvL4Informers().GatewayInformer.Lister().Gateways(namespace).List(labels.NewSelector())
```
[the remark this answers is not in the document]
Comment (abhishekbsingh) [reply internal/nodes/ingress_model_rel.go:998]: all the gateways in 1 namespace are mapped to the same t1LR

File: internal/k8s/advl4_controller.go [hunk 2]
```diff
@@ -204,6 +205,73 @@ func (c *AviController) SetupNamespaceDeletionEventHandler(numWorkers uint32) {
...
+	numWorkers := queue.NumWorkers
+
+	namespaceNetworkInfoHandler := cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			utils.AviLog.Debugf("Namespace NetworkInfo Add")
+			ns, t1lr, ingCIDRPresent, err := fetchNST1LR(obj)
+			if err != nil {
+				utils.AviLog.Errorf(err.Error())
+			}
+			lib.SharedWCPLister().UpdateNamespaceTier1LrCache(ns, t1lr)
```
Comment (swathinsankaran) [inline internal/k8s/advl4_controller.go:241]: Ideally, In ako, the create/update/delete of mappings are done in the Graph layer.

File: internal/nodes/ingress_model_rel.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (swathinsankaran) [reply internal/nodes/ingress_model_rel.go:998]: got it. Thanks

File: internal/k8s/advl4_controller.go (hunk 2 above)
Comment (abhishekbsingh) [reply internal/k8s/advl4_controller.go]: Moved it to Graph Layer, but also had to move VipNetworkList setter/getter to utils as part of it.

File: internal/k8s/controller.go [hunk 3]
```diff
@@ -1262,6 +1262,8 @@ func (c *AviController) Start(stopCh <-chan struct{}) {
 		informersList = append(informersList, lib.AKOControlConfig().AdvL4Informers().GatewayClassInformer.Informer().HasSynced)
 		go lib.AKOControlConfig().AdvL4Informers().GatewayInformer.Informer().Run(stopCh)
 		informersList = append(informersList, lib.AKOControlConfig().AdvL4Informers().GatewayInformer.Informer().HasSynced)
+		go c.dynamicInformers.VCFNetworkInfoInformer.Informer().Run(stopCh)
```
Comment (akshayhavile) [inline internal/k8s/controller.go:1265]: should it be with `if IsVCFCluster()`  condition?

Comment (abhishekbsingh) [reply internal/k8s/controller.go]: yes, thanks

Comment (abhishekbsingh) [reply internal/k8s/controller.go]: yes, thanks

**Review by akshayhavile — APPROVED** [review]

**Pull request merged**